Firestore/Swift/Source/PropertyWrapper/FirestoreQuery.swift

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2021 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+import SwiftUI
+
+public enum FIRPredicate {
```

Comment:
@peterfriese Should we rename this enum as well?

Reply:
Yes, I suggested `QueryPredicate` in https://github.com/chFlorian/firebase-ios-sdk/pull/1/commits/41114476945ce040df34e26c6113d35b7759eef3#diff-78e468ae8bbbcd59d12162d4ba9b4399e9a4aa2c48898f52cb83bbdb7a1a6c79R20